gui/default/syncthing/core/syncthingController.js

```diff
@@ -1402,6 +1402,11 @@ angular.module('syncthing.core')
                 autoNormalize: true
             };
             $scope.editingExisting = false;
+            $scope.newIgnores = {
+                edited: false,
```

Comment:
Well at the point you press save things are set it stone. You could easily add folder paused, set ignores and unpause the folder as part of the addition process